airgun/entities/cloud_insights.py

```diff
@@ -49,6 +60,93 @@ def update(self, values):
         view.fill(values)
 
 
+class RecommendationsTabEntity(BaseEntity):
+    endpoint_path = '/foreman_rh_cloud/insights_cloud'
+
+    def search(self, value):
+        """Search for 'query' and return matched hostnames/recommendations.
+
+        :param value: text to filter (default: no filter)
+        """
+        view = self.navigate_to(self, 'All')
+        time.sleep(5)
```

Comment:
It would be preferable to use a wait_for construct on something you expect to only be present when the page is loaded instead of the sleeps here, I wouldn't block on this but it will likely save you time in the future.

Reply:
I actually tried used a wait_for construct here but the elements inconsistency on the page made time.sleep() the only solution that worked consistently

Reply:
Is that because this is trying to work with both IoP and non-IoP recommendations page? I would recommend against that if possible, since they're totally different